### PR TITLE
Fixed some issues with dsyms for mysql

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

**Comment by mitsuhiko** [comment]
@mattrobenolt that makes the dsyms command work for mysql and the importing as well. Might make sense to also cherry-pick into the next bugfix release.

File: src/sentry/runner/commands/dsym.py [hunk 1]
```diff
@@ -136,6 +137,10 @@ def import_system_symbols(bundles, threads, trim_symbols, no_demangle):
     preprocessed.
     """
     import zipfile
+    from sentry.utils.db import is_mysql
+    if threads != 1 and is_mysql():
+        print >> sys.stderr, 'warning: disable threading for mysql'
```
Comment (dcramer) [inline src/sentry/runner/commands/dsym.py]: should we just use warnings.warn for this

**Comment by mitsuhiko** [comment]
@dcramer fixed. Good to merge?

**Comment by dcramer** [comment]
:+1:

**Pull request merged**